### DISCUSSION TITLE
Need to create standard list for academic degrees offered

### DISCUSSION THREAD

**Discussion by ravi-prakash-v** [beckn/DSEP-Specification#62] [Q&A]
Educational qualifications are used almost everywhere during discovery and application of jobs, courses, mentors, and scholarships - especially Academic Degrees. 

To allow faster discovery and interoperability, there is a need for a standard set of machine-readable codes to be maintained and used in the DSEP ecosystem.

Can anyone recommend any standardized list of degrees offered by academic institutions. If no such standard list exists, then we will need to create one and maintain it.

**Comment by shuklaharsha** [comment]
Following links might help to create one.
https://www.classcentral.com/report/india-online-degrees/
https://targetstudy.com/degree/

**Comment by shaunaksirodaria** [comment]
[1061840_specification-of-degrees-july-2014.pdf](https://github.com/beckn/DSEP-Specification/files/9564660/1061840_specification-of-degrees-july-2014.pdf)
[7914817_BPES-and-MPES.pdf](https://github.com/beckn/DSEP-Specification/files/9564665/7914817_BPES-and-MPES.pdf)
Ref: https://www.ugc.ac.in/ugc_notices.aspx
